<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_title">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>BUBBLE SORT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="6">
            <widget class="QPushButton" name="bt_confirm">
             <property name="text">
              <string>確認</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLineEdit" name="lineEdit_i4"/>
           </item>
           <item row="1" column="4">
            <widget class="QLineEdit" name="lineEdit_o3"/>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="lineEdit_i1"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>輸入數字</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="lineEdit_o4"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>排序後</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_o2"/>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="lineEdit_o1"/>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_i2"/>
           </item>
           <item row="0" column="4">
            <widget class="QLineEdit" name="lineEdit_i3"/>
           </item>
           <item row="1" column="6">
            <widget class="QPushButton" name="bt_clear">
             <property name="text">
              <string>清除</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
